src/textual_web/terminal_session.py

```diff
@@ -48,6 +45,8 @@ async def open(self, width: int = 80, height: int = 24) -> None:
         self.pid = pid
         self.master_fd = master_fd
         if pid == pty.CHILD:
+            os.environ["TERM_PROGRAM"] = "textual-web"
```

Comment:
Yes! I was “surprised” this works because forking is completely new to me. But yes, I ran `textual-web --terminal` and had a `log.debug` to check the value of TERM_PROGRAM.
In the terminal where I ran `textual-web`, it printed `iTerm.app`.
On the browser, it showed `textual-web`.

Should I test this in some other way?

Reply:
I did some research to figure out how this forking works, so I do believe I implemented this in the correct way.